build.gradle

```diff
@@ -4,7 +4,7 @@ apply plugin: 'com.vanniktech.android.junit.jacoco'
 
 buildscript {
     ext.versions = [
-        testRules: '0.5',
+        testRules: '1.0.0',
```

Comment:
You are right, at first it did not work. Managed to fix with my latest commit though.